.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+
+on:
+  push:
+    branches:
+    - master
+    - '*rc'
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  tests:
+    name: Python ${{matrix.python-version}}
+    runs-on: ubuntu-20.04
```

Comment:
Nice, did not really use github actions thus far. Can we also test under windows or OSX with this?

Reply:
Yes, both macOS and Windows Images are available but will count more (x10 and x2)towards the available build minutes. I will integrate those and see

Reply:
Added Windows and macOS CI runs